package.json

```diff
@@ -33,5 +20,27 @@
     "Fabien O'Carroll <fabien@allou.is>",
     "Kirill Fomichev <fanatid@ya.ru>"
   ],
-  "license": "MIT"
+  "files": [
+    "index.js"
```

Comment:
You bring up a good point. It seems like `npm` workflow forces the developer to not publish things to `npm` to support that use case instead of selectively installing "production" things. For example, I'd expect to still publish everything to `npm` and then when someone executes `npm install --production` to only get the "production" things listed in `files`.